vignettes/manual/Constraints.Rmd

```diff
@@ -36,6 +36,9 @@ then the model's ocean carbon uptake (in Pg C) will follow this contents of this
 Ca_constrain=csv:path/to/constraint/file.csv
 ```
 
-then the model's atmospheric carbon pool (given in ppmv CO2) will follow this contents of this file (which should have two columns, `Date` and `Ca_constrain`). The model will extrapolate within this data as necessary, but not beyond its endpoints. If the constraint [CO2] is lower than the calculated [CO2] at that time step, any excess is moved to the deep ocean; conversely, any necessary deficit is transferred from there.
+then the model's atmospheric carbon pool (given in ppmv CO2) will follow this contents of this file (which should have two columns, `Date` and `Ca_constrain`). If the constraint [CO2] is lower than the calculated [CO2] at that time step, any excess is moved to the deep ocean; conversely, any necessary deficit is transferred from there.
+Otherwise Hector's atmospheric carbon pool can be set using the `setvars` function and `CA_CONSTRAIN` variable.
```

Comment:
Same here.

Reply:
Blegh -- good idea to read all comments before committing. Addressed in 88a78e9.

Reply:
I feel your pain.  I just submitted the same set of revisions 4 (!) times because I kept finding minor mistakes and typos right at the final review stage.